tests/e2e/flowtest/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,239 @@
+            sa_name=sa_name.name,
+            dc_deployment=True,
+        )
+        helpers.wait_for_resource_state(
+            resource=pod_obj, state=constants.STATUS_RUNNING
+        )
+        return pod_obj, pvc_obj
+
+    def copy_files(self, pod_obj, pvc_obj):
+        log.info(f"pod obj name---- {pod_obj.name}")
+        pod_name = pod_obj.name
+
+        log.info("cephfs pod created")
+        ocp_obj = ocp.OCP(
+            kind=constants.POD,
+            namespace=pvc_obj.project.namespace,
+        )
+        # Number of times we copy the kernel
+        copies = 3
+        tmploc = ntar_loc.split("/")[-1]
+        ocp_obj.exec_oc_cmd(
+            f"rsync {ntar_loc} {pod_name}:{constants.FLEXY_MNT_CONTAINER_DIR}",
+            timeout=300,
+        )
+        ocp_obj.exec_oc_cmd(
+            f"exec {pod_name} -- mkdir {constants.FLEXY_MNT_CONTAINER_DIR}/x"
+        )
+        for x in range(copies):
+            ocp_obj.exec_oc_cmd(
+                f"exec {pod_name} -- /bin/tar xf"
+                f" {constants.FLEXY_MNT_CONTAINER_DIR}/{tmploc}/{TARFILE}"
+                f" -C {constants.FLEXY_MNT_CONTAINER_DIR}/x/x{x}",
+                timeout=3600,
+            )
+        log.info("cephfs test files created on pod")
+
+    def calculate_md5sum(self, pod_obj, pvc_obj):
+        pod_name = pod_obj.name
+        data_path = f'"{constants.FLEXY_MNT_CONTAINER_DIR}/x"'
+        ocp_obj = ocp.OCP(
+            kind=constants.POD,
+            namespace=pvc_obj.project.namespace,
+        )
+        random_file = ocp_obj.exec_oc_cmd(
+            f"exec -it {pod_name} -- /bin/bash"
+            f' -c "find {data_path} -type f | "shuf" -n 1"',
+            timeout=300,
+        )
+
+        md5sum_pod_data = pod.cal_md5sum(pod_obj=pod_obj, file_name=random_file)
+        return md5sum_pod_data
+
+    def data_integrity_check(self, pod_obj, pvc_namespace):
+        pod_name = pod_obj.name
+        log.info(f"pod obj name---- {pod_name}")
+        ocp_obj = ocp.OCP(
+            kind=constants.POD,
+            namespace=pvc_namespace,
+        )
+
+        data_path = f"{constants.FLEXY_MNT_CONTAINER_DIR}/x"
+        random_file = ocp_obj.exec_oc_cmd(
+            f"exec -it {pod_name} -- /bin/bash"
+            f' -c "find {data_path} -type f | "shuf" -n 1"',
+            timeout=300,
+        )
+        print(f"files are {random_file}")
+        ini_md5sum_pod_data = pod.cal_md5sum(pod_obj=pod_obj, file_name=random_file)
+        start_time = time.time()
+        assert validate_pods_are_respinned_and_running_state([pod_obj])
+        end_time = time.time()
+        total_time = end_time - start_time
+        log.info(f"Time taken by pod to restart is  {total_time}")
+        pod_objs = pod.get_all_pods(
+            namespace=pvc_namespace, selector=["pod-test-rbd"], selector_label="name"
+        )
+        for pod_obj in pod_objs:
+            pod_name = pod_obj.name
+            if "-1-deploy" not in pod_name:
```

Comment:
the main intent of the function here is to cal the data_integrity, This code doesnt seems relevant to this function, please move it outside of this